src/lib/skilling/skills/herblore/mixables/grimy.ts

```diff
@@ -0,0 +1,134 @@
+import itemID from '../../../../util/itemID';
+import { Mixable } from '../../../types';
+import { transformStringBankToNum } from '../../../../util/transformStringBankToNum';
+
+const Grimy: Mixable[] = [
+	{
+		name: 'Guam leaf',
+		id: itemID('Guam leaf'),
+		level: 1,
+		xp: 2.5,
+		inputItems: transformStringBankToNum({ 'Grimy guam leaf': 1 }),
+		tickRate: 1,
```

Comment:
i would advocate for a 0.5 tick rate for these since its trivial to click twice in a single tick and i think that can be expected of from a minion.